documentation/library_guide/parallel_api/pass_data_algorithms.rst

```diff
@@ -81,7 +81,20 @@ the buffer were created for the same queue. For example:
     return 0;
   }
 
-Alternatively, use ``std::vector`` with a USM allocator. For example:
+Alternatively, use ``std::vector`` with a USM allocator.
+
+Note: The ability to appropriately detect USM allocated ``std::vector::iterator`` depends
+on details of the C++ standard library implementation and what information about the
+allocator is included in the ``std::vector::iterator`` type definition. If USM allocated
+vector iterators are not detectable with your C++ standard library, they will still function
+as inputs to oneDPL, but they will be treated as if they were host-side
+``std::vector::iterator`` as described in the `Use Host-Side std::vector`_ section. To guarantee
+no additional host-side copies, you can use ``std::vector::data()`` in combination with
+``std::vector::size()`` with USM allocated vectors to obtain USM pointers which can be used as
```

Comment:
Please add a comma between USM pointers and which.

Reply:
thanks, fixed.